core/src/main/java/org/apache/iceberg/util/SnapshotUtil.java

```diff
@@ -149,8 +149,7 @@ public static Iterable<Snapshot> ancestorsOf(long snapshotId, Function<Long, Sna
   }
 
   /**
-   * Traverses the history of the table's current snapshot and finds the first snapshot committed
-   * after the given time.
+   * Finds the oldest snapshot of a table that was committed either at or after a given time.
```

Comment:
@flyrain, this change is incorrect. This is not the oldest known snapshot, it is the oldest known snapshot in the current snapshot's history. Can you fix it?

Reply:
Sure, will do.

Reply:
Filed https://github.com/apache/iceberg/pull/6479